service/storage_service.cc

```diff
@@ -2374,6 +2374,7 @@ class topology_coordinator : public endpoint_lifecycle_subscriber {
 
         switch (*tstate) {
             case topology::transition_state::join_group0: {
+                rtlogger.info("entered `join_group0` transition state");
```

Comment:
Why can't you log it once before the switch:  `rtlogger.info("entered {} transition state", topology::transition_state::join_group0);`

Reply:
Right, fixed